src/main/java/org/apache/sysds/runtime/controlprogram/paramserv/ParamServer.java

```diff
@@ -119,7 +175,7 @@ public ListObject getResult() {
 		// so we could return directly the result model
 		return _model;
 	}
-	
+
 	protected synchronized void updateGlobalModel(int workerID, ListObject gradients) {
 		try {
 			if (LOG.isDebugEnabled()) {
```

Comment:
everything is surrounded by a is debug enabled witch is a lower level than info.